### DISCUSSION TITLE
Add innactive cells to Local Grid Refinement

### DISCUSSION THREAD

**Discussion by laura-del-val-Eurecat** [modflowpy/flopy#2360] [Q&A]
Hi Everyone!

I have a question: I'm trying to generate a grid using the LGR utility, but I don't know in which grid (the parent or the child) and how (giving 1 or 0 value) should I specify the inactive cells. This is a model of a catchment (so it is NOT rectangular ) were cells around the catchment limits should be inactive.

Thank you in advance for the help. Cheers!!

**Comment by christianlangevin** [comment]
Hello @laura-del-val-Eurecat, there is some documentation [here](https://flopy.readthedocs.io/en/3.4.1/Notebooks/lgr_tutorial01.html), which you have probably seen already.  The idea is that you create an indicator array for the parent grid.  In the example, this array is called `idomainp`. This array should have a 1 everywhere except for cells that you want to be represented with the child grid.  This indicator array is then passed to to the `Lgr` utility.  In your case you also want to deactivate other parts of the model in order to follow the boundary of your area of interest.  You will need to do that as a separate option after you have used the `Lgr` utility.  

In the example, the line below creates the discretization package for the parent model:

```
dis = flopy.mf6.ModflowGwfdis(gwfp, **lgrp.get_gridprops_dis6())
```

You'll need to modify that and instead pass in an idomain (shown below as `myidomainarray`) that you create, which has 0's in the area for the child grid and also 0's in the inactive part of your model domain.  The trick for you will be figuring out how to create `myidomainarray`, which hopefully can be done using flopy intersect utilities or something else.

```
dis_dict = lgrp.get_gridprops_dis6()
dis_dict["idomain"] = myidomainarray
dis = flopy.mf6.ModflowGwfdis(gwfp, )
```
Hopefully this can help keep you moving.

**Comment by nash87** [comment]
## v3.5.0 Released

Three new features shipped:

1. **Visitor Pre-Registration** — Pre-register visitors with QR code passes, check-in tracking, admin overview (#230)
2. **EV Charging Stations** — Manage chargers (Type2/CCS/CHAdeMO/Tesla), start/stop sessions, kWh tracking (#231)
3. **Smart Recommendations** — Weighted scoring (frequency 40%, availability 30%, price 20%, distance 10%), badges in booking flow (#232)

Release: https://github.com/nash87/parkhub-rust/releases/tag/v3.5.0
